inspect/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package inspect contains utilities for extracting information about a
```

Comment:
This "inspect" package goes against the design. Any information that is needed to produce reports should be added to the metadata when constructing the test configuration. This is already the case for scenario. Any other informatioin that is needed can be added by the loadtest config generator, either explicitly as a flag or by updating the code to generate it (as already happens with scenario).

We need a design discussion and review before continuing with the implementation of this feature.